### PR TITLE
[zh-cn] Update 'how browsers work'

### PR DESCRIPTION
**Pull request by jasonren0403**
### Description

Update content and remove one bad img reference

### CODE REVIEW

File: files/zh-cn/web/performance/how_browsers_work/index.md [hunk 1]
```diff
@@ -57,41 +61,41 @@ TCP 的“三次握手”技术经常被称为“SYN-SYN-ACK”——更确切
...
+TCP 数据包在传输过程中被分成若干段。由于 TCP 保证数据包的顺序，因此服务器在发送一定数量的数据包后，必须以 ACK 数据包的形式收到客户端的确认。
 
-在 {{glossary('TCP slow start', "TCP 慢启动")}} 中，在收到初始包之后，服务器会将下一个数据包的大小加倍到大约 28KB。后续的数据包依次是前一个包大小的二倍直到达到预定的阈值，或者遇到拥塞。
+如果服务器在每个网段后都等待 ACK，则会导致客户端频繁发出 ACK，即使在网络负荷较低的情况下也会增加传输时间。
 
-![TCP 慢启动](congestioncontrol.jpg)
+另一方面，一次性发送过多网段可能会导致这样的问题：在繁忙的网络中，客户端无法接收到网段，只能长时间不停地回应 ACK，服务器不得不不断重新发送网段。
 
-如果你听说过初始页面加载的 14KB 规则，TCP 慢启动就是初始响应为 14KB 的原因，也是为什么 web 性能优化需要将此初始 14KB 响应作为优化重点的原因。TCP 慢启动逐渐建立适合网络能力的传输速度，以避免拥塞。
+为了平衡传输段的数量，{{glossary('TCP slow start', 'TCP 慢启动')}} 算法用于逐渐增加传输数据量，直到确定最大网络带宽，并在网络负载较高时减少传输数据量。
```
Comment (yin1999) [inline files/zh-cn/web/performance/how_browsers_work/index.md:94]:
```suggestion
为了平衡传输段的数量，{{glossary('TCP slow start', 'TCP 慢启动')}}算法用于逐渐增加传输数据量，直到确定最大网络带宽，并在网络负载较高时减少传输数据量。
```

**Review by yin1999 — COMMENTED** [review]
Could you also fix the broken link listed [here](https://pr16764.content.dev.mdn.mozit.cloud/zh-CN/docs/Web/Performance/How_browsers_work#_flaws)

File: files/zh-cn/web/performance/how_browsers_work/index.md [hunk 2]
```diff
@@ -136,63 +140,63 @@ DOM 树描述了文档的内容。[`<html>`](/zh-CN/docs/Web/HTML/Element/html)
...
+### 合成
 
 当文档的各个部分以不同的层绘制，相互重叠时，必须进行合成，以确保它们以正确的顺序绘制到屏幕上，并正确显示内容。
 
 当页面继续加载资源时，可能会发生回流（回想一下我们迟到的示例图像），回流会触发重新绘制和重新组合。如果我们定义了图像的大小，就不需要重新绘制，只需要重新绘制需要重新绘制的层，并在必要时进行合成。但我们没有包括图像大小！从服务器获取图像后，渲染过程将返回到布局步骤并从那里重新开始。
 
 ## 交互
 
-一旦主线程绘制页面完成，你会认为我们已经“准备好了”，但事实并非如此。如果加载包含 JavaScript（并且延迟到 [`onload`](/zh-CN/docs/Web/API/GlobalEventHandlers/onload) 事件激发后执行），则主线程可能很忙，无法用于滚动、触摸和其他交互。
+一旦主线程绘制页面完成，你会认为我们已经“准备好了”，但事实并非如此。如果加载包含 JavaScript（并且延迟到 [`onload`](/zh-CN/docs/Web/API/GlobalEventHandlers/onload) 事件触发后执行），则主线程可能很忙，无法用于滚动、触摸和其他交互。
```
Comment (jasonren0403) [inline files/zh-cn/web/performance/how_browsers_work/index.md:197]:
```suggestion
一旦主线程绘制页面完成，你会认为我们已经“准备好了”，但事实并非如此。如果加载包含 JavaScript（并且延迟到 [`onload`](/zh-CN/docs/Web/API/Window/load_event) 事件触发后执行），则主线程可能很忙，无法用于滚动、触摸和其他交互。
```

**Review by yin1999 — APPROVED** [review]
Thanks

**Pull request merged**